distribution/changelog.txt

```diff
@@ -38,6 +38,7 @@
 - Fix: [#17292] Rows in shortcut key list stay highlighted when cursor leaves list.
 - Fix: [#17295] Pause status not cleared when loading a scenario made from a converted paused save.
 - Fix: [#17310] Reversed reversible vehicles not imported properly when loading RCT1 parks.
+- Fix: [#17316] Sides of River Rapids� corners overlay other parts of the track.
```

Comment:
I see a weird question mark icon here on the changelog, am I alone? 🤔

Reply:
Yea it's a `’` but for some reason it shows up as a ?

Reply:
Likely saved with the wrong encoding.

Reply:
I cannot seem to change it, as github doesn't see it as a change. Might just be an issue with the preview, because if i go in it on github's editing page it shows up just fine.

Reply:
Other special characters also appear broken on your branch:

https://github.com/karst/OpenRCT2/blob/f251af2aa6bff680a601911036b4cbb5bd995b24/distribution/changelog.txt#L30

While they are fine on OpenRCT2's develop:

https://github.com/OpenRCT2/OpenRCT2/blob/8e509d9669765dc68523ae82fecc41347b5577ff/distribution/changelog.txt#L30

Try using Notepad++ or some other tool that allows you to easily save with UTF-8 encoding.

Reply:
Yea, very vague as usually it does save my txt files in UTF-8. I was able to fix it.